website/project/views/contributor.py

```diff
@@ -439,21 +440,32 @@ def send_claim_email(email, unclaimed_user, node, notify=True, throttle=24 * 360
     """
 
     claimer_email = email.lower().strip()
-
     unclaimed_record = unclaimed_user.get_unclaimed_record(node._primary_key)
     referrer = User.load(unclaimed_record['referrer_id'])
     claim_url = unclaimed_user.get_claim_url(node._primary_key, external=True)
 
-    # When adding the contributor, the referrer provides both name and email.
-    # The given email is the same provided by user, just send to that email.
+    # Option 1:
+    #   When adding the contributor, the referrer provides both name and email.
+    #   The given email is the same provided by user, just send to that email.
+    preprint_provider = None
     if unclaimed_record.get('email') == claimer_email:
-        mail_tpl = getattr(mails, 'INVITE_{}'.format(email_template.upper()))
+        # check email template for branded preprints
+        if email_template == 'preprint':
```

Comment:
(Add an author who does not have an OSF account) Query the database and find if the project is associated with a preprint. If so, use the customized templates.